src/D2L.CodeStyle.TestAnalyzers/NUnit/ConfigTestSetupStringsAnalyzer.cs

```diff
@@ -73,12 +73,12 @@ INamedTypeSymbol attributeType
 
 			foreach( AttributeSyntax attribute in attributes ) {
 
-				ISymbol symbol = context
+				ISymbol? symbol = context
 					.SemanticModel
 					.GetSymbolInfo( attribute, context.CancellationToken )
 					.Symbol;
 
-				if( symbol.IsNullOrErrorType() ) {
+				if( symbol == null || symbol.Kind == SymbolKind.ErrorType ) {

```

Comment:
Was losing the `TypeKind` check intentional here?

Reply:
👍 I'm going to switch it back to the extension anyway for simplicity (I still dislike them, kind of) using `[NotNullWhen( false )]` as discussed above.